portfolio/src/main/webapp/script.js

```diff
@@ -65,7 +65,15 @@ function grow() {
 }
 
 function getGreeting() {
-  fetch('/data').then(response => response.text()).then(greeting => {
-    document.getElementById('greeting-container').innerHTML = greeting;
+  fetch('/data').then(response => response.json()).then(greetingsJson => {
```

Comment:
Everything in js is a json. Prefer to call the variable `greetings`

Reply:
Got it!